### PR TITLE
feat: big number comp color switch

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4884 

### Description:

For cases where an increase in a KPI metric is not a good thing, the user should be able to switch what 'green' represents. By default, red = bad and green = good. 
- [x] Add colour to the comparison number and arrow.  By default, `arrow-down = red6`  and `arrow-up = green6`
- [x] Add a toggle in the config panel: `Flip colors`
- [x] If the user turns the toggle on, `arrow-down = green6`  and `arrow-up = red6`
- [x] If there is no change, the label colour remains unaffected.

https://github.com/lightdash/lightdash/assets/67699259/e9991323-ceac-4fd0-a939-33accb74e899

### CODE REVIEW

File: packages/frontend/src/components/SimpleStatistic/index.tsx [hunk 1]
```diff
@@ -161,7 +168,9 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
                             marginTop: 10,
                             display: 'flex',
                             alignItems: 'center',
-                            color: comparisonValueColor,
+                            color: `${
+                                flipColors ? comparisonValueColor : 'inherit'
```
Comment (ZeRego) [inline packages/frontend/src/components/SimpleStatistic/index.tsx:172]: should we check the flip logic in `comparisonValueColor` ? 
```
            case ComparisonDiffTypes.POSITIVE:
                return flipColors ? Colors.RED3 : Colors.GREEN3;
            case ComparisonDiffTypes.NEGATIVE:
                return flipColors ? Colors.GREEN3 : Colors.RED3;
```

Comment (yasmin-e) [reply packages/frontend/src/components/SimpleStatistic/index.tsx:172]: I completely misunderstood the requirements 🥲 will re-work this

Comment (ZeRego) [reply packages/frontend/src/components/SimpleStatistic/index.tsx:172]: eg: in this case the comparison value should be red

Captura de ecrã 2023-05-12, às 10 39 58

Comment (yasmin-e) [reply packages/frontend/src/components/SimpleStatistic/index.tsx:172]: got it!

File: packages/frontend/src/hooks/useBigNumberConfig.ts [hunk 2]
```diff
@@ -203,6 +206,7 @@ const useBigNumberConfig = (
         setComparisonFormat(
             bigNumberConfigData?.comparisonFormat ?? ComparisonFormatTypes.RAW,
         );
+        setFlipColors(bigNumberConfigData?.flipColors ?? true);
```
Comment (ZeRego) [inline packages/frontend/src/hooks/useBigNumberConfig.ts:209]: Let's have `false` as default value. 

false = positive green / negative red
true = prositive red / negative green

**Review by ZeRego — APPROVED** [review]
🎉

**Pull request merged**